Dockerfile

```diff
@@ -181,6 +192,9 @@ COPY hack/scripts/cleanup.sh /bin
 RUN cleanup.sh /rootfs
 COPY hack/scripts/symlink.sh /bin
 RUN symlink.sh /rootfs
+# bootloader
+COPY --from=base /rootfs/share/gptmbr.bin /rootfs/usr/share/gptmbr.bin
+COPY --from=base /rootfs/bin/extlinux /rootfs/bin/extlinux
```

Comment:
I think we can drop this line. We already do a `COPY --from=base /rootfs ./` a few steps before.